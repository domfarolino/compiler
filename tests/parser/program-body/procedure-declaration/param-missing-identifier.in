program dominic is
  procedure domFunc(string inout);
begin end program.
