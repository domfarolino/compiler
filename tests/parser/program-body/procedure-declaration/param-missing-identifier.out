[1;31mLine 2: Expected identifier in variable declaration, but got: 'inout'
Line 2: Error parsing program body
[0m